app/controllers/idv/in_person_controller.rb

```diff
@@ -16,7 +16,7 @@ class InPersonController < ApplicationController
 
     FLOW_STATE_MACHINE_SETTINGS = {
       step_url: :idv_in_person_step_url,
-      final_url: :idv_in_person_proofing_ssn_url,
+      final_url: :idv_in_person_verify_info_url,
```

Comment:
@svalexander  I think this should be `idv_in_person_proofing_ssn_url`. Check out my delete [PR](https://github.com/18F/identity-idp/pull/8534/files) for Verify. The final url is actually the first non FSM step

Reply:
yes, you're right! i updated